frontend/public/module/k8s/docker.js

```diff
@@ -37,8 +37,10 @@ export const getContainerState = function(containerStatus) {
   }
 
   const stateKey = keys[0];
-  state.label = stateKey;
   _.assign(state, containerStatus.state[stateKey]);
+  state.label = _.includes(stateKey, 'terminated') && _.isFinite(state.exitCode)
```

Comment:
Actually don't you want `===` instead of `_.includes`?

```suggestion
  state.label = stateKey === 'terminated' && _.isFinite(state.exitCode)
```